store/postgres/src/transaction_receipt.rs

```diff
@@ -0,0 +1,166 @@
+use diesel::{
+    pg::{Pg, PgConnection},
+    prelude::*,
+    query_builder::{Query, QueryFragment},
+    sql_types::{Binary, Nullable},
+};
+use diesel_derives::{Queryable, QueryableByName};
+use itertools::Itertools;
+use std::convert::TryFrom;
+use web3::types::H256;
+
+use graph::prelude::transaction_receipt::LightTransactionReceipt;
+
+/// Queries the database for all the transaction receipts in a given block range.
+pub fn find_transaction_receipts_in_block(
+    conn: &PgConnection,
+    blocks_table_name: &str,
+    block_hash: H256,
+) -> anyhow::Result<Vec<LightTransactionReceipt>> {
+    let query = TransactionReceiptQuery {
+        blocks_table_name,
+        block_hash: block_hash.as_bytes(),
+    };
+
+    query
+        .get_results::<RawTransactionReceipt>(conn)
+        .or_else(|error| {
+            Err(anyhow::anyhow!(
+                "Error fetching transaction receipt from database: {}",
+                error
+            ))
+        })?
+        .into_iter()
+        .map(LightTransactionReceipt::try_from)
+        .collect()
+}
+
+/// Parameters for querying for all transaction receipts of a given block.
+struct TransactionReceiptQuery<'a> {
+    block_hash: &'a [u8],
+    blocks_table_name: &'a str,
+}
```

Comment:
Since this query is very simple, it would have been enough to just use `sql_query` for this; but since it's there, it's fine to leave it as is. My rule of thumb is that if a query can be easily written as one string (maybe with using `format!` to get a table name or similar in there) I use `sql_query`, and only when query generation gets more complex than that do I resort to the whole `QueryFragment` dance.